.circleci/config.yml

```diff
@@ -125,7 +125,7 @@ jobs:
           command: |
             hack/go-generate.sh
             go mod download
-            golangci-lint run
+            golangci-lint run --build-tags fuse_cli,fsintegration
```

Comment:
didn't we start the lint rc file, `.golangci.yml` because we think separation of concerns vis
* linter config
and
* ci config
 is desirable?

Reply:
good catch. Moving this to config